modules/gitops-release-notes-1-15-2.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-15.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-15-2_{context}"]
+= Release notes for {gitops-title} 1.15.2
+
+{gitops-title} 1.15.2 is now available on {OCP} 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-15.2_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-15-2-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.15.2 security update advisory
+
+Issued: 2025-MM-DD
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-15-2_{context}"]
+== New features
+
+* With this update, a new check box, `.spec.SyncPolicy.Automated`, has been added to the Argo CD Application resource to explicitly enable or disable `automatedSync`. This option provides the following benefits:
```

Comment:
Did we want to go with _option_ or _parameter_ over check box, [as discussed](https://github.com/openshift/openshift-docs/pull/92626#discussion_r2077501740)?

Reply:
I think I will go with "Option" as per your suggestion @mburke5678.